components/BookCard.js

```diff
@@ -14,9 +14,17 @@ export default function BookCard({ title, author, onDelete }) {
   return (
     <Container>
       <Card>
+        <CoverContainer>
+          <Image
+            src={`${cover}`}
```

Comment:
this should also work like this:
```suggestion
            src={cover}
```